components/dropdown/dropdown.tsx

```diff
@@ -5,7 +5,7 @@ import DropdownButton from './dropdown-button';
 import warning from '../_util/warning';
 
 export interface DropDownProps {
-  trigger?: ('click' | 'hover')[];
+  trigger?: ('click' | 'hover'| 'contentMenu')[];
```

Comment:
contextMenu

Reply:
You are right! However https://ant.design/components/dropdown-cn/ is still contentMenu. I was misled by it